src/ducks/transactions/TransactionModal.styl

```diff
@@ -11,9 +11,16 @@
     border-top 1px solid var(--silver)
 
 .TransactionModalRow
+    min-height 57px
+
+    & + &
+        border-top 1px solid var(--silver)
```

Comment:
Question: I am more accustomed to using

```
.TransactionModalRow
  &:last-child
    border: 0
```

Is there a difference ?

Reply:
Visually, both techniques are usually used to achieve the same goal